RELEASE_NOTES.md

```diff
@@ -1,5 +1,83 @@
+  `transport::socket` module.
+* Rename `RawTransport` to `TcpTransport` to better reflect the underlying capability.
+  Also rename the `socket::raw` module to `socket::tcp`.
+* Add a `comments` field to the `Circuit` object to allow for human-readable comments.
+* Save a disconnected connection in a `HashMap` to be returned in case the connection
+  is removed again at a later time.
+* Separate circuit and proposal REST API responses from the internal structs so
+  the internal structures can change as needed without impacting the data exposed
+  via the REST API.
+* Add a `BiHashMap` to `Mesh` for a unique ID to the mesh ID.
+* Allow `BiHashMap` look-ups with elements that implement the `Borrow` trait.
+* Replace the `RecvError::InternalError` with a `Disconnected` error for accuracy.
+* Experimental feature “circuit-template” (new): Implement `CircuitTemplateManager`
+  with functionality to list and load available templates.
+* For all experimental Biome features:
+    - Remove Arc wrappings from Biome’s user store to allow mutable references of
+      the user store.
+    - Separate Biome models and schemas into their respective modules.
+    - Add a `/biome/verify` REST API endpoint to verify a user’s password.
+    - Update the Biome Rust API documentation to remove explicit references to
+      Rust features.
+    - Make the Biome user module private.
+    - Rename `biome::datastore` to `biome::migrations` and add module-level
+      documentation for biome migrations.
+    - Remove all uses of `super` imports in Biome.
+* Correct `json-web-tokens` feature guards.
+* Experimental feature “proposal-read”:
+    - Update the `ProposalStore::proposals` arguments to take a `ProposalFilter`
+      enum to allow the proposal store to filter returned proposals.
+    - Rename the `/admin/proposals` endpoint's `filter` query parameter to `management_type`.
+    - Add `member` filter query parameter to the GET `/admin/proposals/` REST API endpoint.
+
+### CLI:
+* Experimental feature “circuit”:
+    - Update the `splinter circuit create` command to generate a service ID automatically.
+    - Add a `comments` argument to the `splinter circuit create` command to optionally
+      provide comment for the proposed circuit.
+    -  Display comments in the output of the `splinter proposals` command.
+    - Update the `circuit-read` and `proposal-read` response deserializers to reflect
+      the REST API response objects, rather than the related internal structs.
+    - Add a `member` argument to the `splinter circuit proposals` command for
+      filtering proposals by the given member.
+    - Add `SPLINTER_REST_API_URL` environment variable to be used if `-U` or `--url`
+      is not specified.
+    - Remove the `required` specification from `splinter circuit list` and
+      `splinter circuit proposals` arguments that are not actually required.
+* Rename the `DEFAULT_ENDPOINT` constant to `DEFAULT_SPLINTER_REST_API_URL`.
```

Comment:
This probably isn't necessary to mention since it doesn't change any functionality